core-web/libs/edit-content/src/lib/utils/functions.util.ts

```diff
@@ -113,3 +114,59 @@ export const transformLayoutToTabs = (
 
     return tabs;
 };
+
+/**
+ * Checks if a given value is a valid JSON string
+ *
+ * @param {string} value - The value to be checked
+ * @returns {boolean} - True if the value is a valid JSON string, false otherwise
+ */
+export const isValidJson = (value: string): boolean => {
+    try {
+        const json = JSON.parse(value);
+
+        return json !== null && typeof json === 'object' && !Array.isArray(json);
+    } catch (e) {
+        console.warn(`${value} is not a valid JSON`);
```

Comment:
If the user look and the console and this is the only error they see, they will not know what to fix, I think we can do better here, specially for this `fieldVariable` options

Reply:
It return a boolean. That can be handled direct in the origin I think.